src/sentry/static/sentry/app/components/events/interfaces/frame/defaultTitle/index.tsx

```diff
@@ -12,9 +12,9 @@ import {t} from 'app/locale';
 import {getMeta} from 'app/components/events/meta/metaProxy';
 import space from 'app/styles/space';
 
-import FrameFunctionName from './frameFunctionName';
-import {getPlatform, trimPackage} from './utils';
-import FrameDefaultTitleOriginalSourceInfo from './frameDefaultTitleOriginalSourceInfo';
+import FrameFunctionName from '../functionName';
```

Comment:
```suggestion
import FunctionName from '../functionName';
```
Should this also be renamed?